src/Type/Schema.php

```diff
@@ -339,15 +341,22 @@ private function loadType(string $typeName) : ?Type
 
         $type = $typeLoader($typeName);
 
+        if (is_callable($type)) {
+            $type = self::resolveType($type);
```

Comment:
```suggestion
            $type = $type();
```

Reply:
Nevermind, i think we can remove the `is_callable()` check here.

Reply:
>  we can remove the is_callable() check her

I would love that, and if this were my personal code base it would already be done, but we have a lot of (sort of bizarre) tests that want to demonstrate what happens when bad types are passed through the system, and making that change would cause them to fail and open up a huge can of worms.

Not to sound like a broken record, but in my next PR I'm going to overhaul all of this, and while I'm at it I'll see if I can get away with killing those tests.

Reply:
https://github.com/webonyx/graphql-php/pull/557/files/72ccaa301778869cafbd35d2ce6746b21872b4cf#r439629614